### PR TITLE
New package: RustDesk.RustDesk.Nightly version 1.2.4

### PR DESCRIPTION
**Pull request by rakleed**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132925)

### CODE REVIEW

File: manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  Interactive: --install
+UpgradeBehavior: install
+ReleaseDate: 2024-01-05
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.4-x86-sciter.exe
+  InstallerSha256: E64A1792446002F13628CEDF1BB0517A0E4A6FFA8799028D25A2F304A6117D44
+- Architecture: x64
+  InstallerUrl: https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.4-x86_64.exe
+  InstallerSha256: C9B40A905F3E778E6BD7646039CF2C8077C13E6FBBF74A70080884A5C23C780B
```
Comment (OfficialEsco) [inline manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml:20]: At this time the hash is sadly outdated

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml (hunk 1 above)
Comment (rakleed) [reply manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml]: @OfficialEsco they update nightly builds once a day with all the latest commits

**Review by OfficialEsco — APPROVED** [review]

File: manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/r/RustDesk/RustDeskNightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml]: Ye which could be problematic if there is delay with the bot or the mods ^^

File: manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: RustDesk.RustDeskNightly
```
Comment (OfficialEsco) [inline manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.installer.yaml:4]:
```suggestion
PackageIdentifier: RustDesk.Nightly
```

File: manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: RustDesk.RustDeskNightly
```
Comment (OfficialEsco) [inline manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: RustDesk.Nightly
```

File: manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: RustDesk.RustDeskNightly
```
Comment (OfficialEsco) [inline manifests/r/RustDesk/Nightly/1.2.4/RustDesk.RustDeskNightly.yaml:4]:
```suggestion
PackageIdentifier: RustDesk.Nightly
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\r\RustDesk\Nightly\1.2.4', expected 'manifests\r\RustDesk\RustDeskNightly\1.2.4'

Ahh you updated the folder path to a more reasonable one, just gotta update the PackageIdentifier to match ^^
OH and the Manifest names!

**Review by OfficialEsco — APPROVED** [review]

**Comment by rakleed** [comment]
@OfficialEsco did I choose the correct name for PackageID and path nesting? I searched the documentation and GitHub repository, but did not find any mention of how to add beta/preview/nightly versions, so I followed the example of [Windows Terminal](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/WindowsTerminal).

**Comment by OfficialEsco** [comment]
Yepyep looks 👌 now

**Comment by rakleed** [comment]
@OfficialEsco @stephengillie what's wrong with the bot?

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
Can't remember what @stephengillie does but its probably a rerun..

**Pull request merged**